docs/development/releasing.rst

```diff
@@ -442,6 +460,82 @@ be fixed soon to a new release milestone.  If the upcoming bug fix release is
 issues that you don't expect to be fixed in time for 'v0.2.2'.
 
 
+.. _helpers-release-info:
+
+Coordinating Astropy and astropy-helpers Releases
+-------------------------------------------------
+
+A bit more initial effort is required for an Astropy release that has a
+corresponding astropy-helpers_ release.  The main reason for this more complex
+procedure is to allow the Astropy core to be tested against the new helpers
+before anything is released.  Hence the following procedure should be added
+to the beginning of the above procedure when this is required. This procedure
+applies both for regular release *and* release candidates are the same
+(except that version numbers have ``rc#`` at the end).
+
+#. In the `astropy-helpers repository`_, create a new (temporary) branch
+   "tmp-release-v<version>"::
+
+      $ cd /wherever/you/put/astropy/astropy_helpers
+      $ git branch tmp-release-v<version> <maintenance branch name>
+
+#. In that branch, create a release commit by updating the version info and
+   changelog as described in the release instructions above.
+
+#. Push the branch you just created to the `astropy-helpers repository`_ on
+   github::
+
+      $ git push upstream tmp-release-v<version>
+
+#. In astropy master (or the relevant maintenance branch for the release you
+   are doing), issue a PR updating the helpers to the commit described in the
+   last step (i.e., the commit at the head of the "tmp-release-v<version>"
+   branch you just created).  The easiest way to do this is::
+
+      $ cd /wherever/you/put/astropy
+      $ cd astropy_helpers
+      $ git fetch upstream  # you probably did this already in the previous step
+      $ git checkout upstream/tmp-release-v<version>
+      $ cd ..
+      $ git add astropy_helpers
+      $ git commit -m "updated helpers to v<version>"
+
+#. Wait for the continuous integration services (E.g., Travis) to run on the PR
+   to ensure the release commit of the helpers works with the to-be-released
+   version of Astropy.
+
+#. If the PR's tests fail, fix whatever the problem is, and then re-do this
+   procedure. You'll need to either delete the previous "tmp-release-v<version>"
+   branch on the github `astropy-helpers repository`_ or use ``git push -f``
+   when you push up the replacement temporary release branch. You can re-use the
+   PR into the `astropy core repository`_ (created in the step just before this
+   one) by updating the ``astropy_helpers`` submodule to point to the new
+   "tmp-release-v<version>" from  *after* the fix - that way you don't need to
+   make another PR for the fixed version.
+
+#. Once the tests all succeed, finish the release of the helpers by doing this
+   in the helpers repo::
+
+      $ git checkout <maintenance branch name>
+      $ git merge --no-ff tmp-release-v<version>
+      $ git tag -s "v<version>" -m "Tagging v<version>"
+      $ git push upstream --tags <maintenance branch name>
```

Comment:
Do we need to also mention the ``python setup.py sdist register upload``? Or just refer to the instructions for the core package?